etc/ci.sh

```diff
@@ -46,10 +48,20 @@ make
 # return to base directory
 popd
 
-# create dir for coverage results
-COVDIR=coverage
-mkdir -p $COVDIR
-
+if [[ "${GEN_COVERAGE}" == yes ]]
```

Comment:
If you do this, you also need to adjust `.appveyor.yml` to set `GEN_COVERAGE`, else it will stop producing coverage reports.

Also, when using `[[ ]]` in bash, there is no need to put quotes around the variable name.